source-kinesis/Dockerfile

```diff
@@ -0,0 +1,42 @@
+# Build Stage
+################################################################################
+FROM golang:1.17-buster as builder
+
+WORKDIR /builder
+
+# Compile-time dependencies
+# RUN apt-get update \
+#  && apt-get install -y \
+#       ca-certificates \
+#  && apt-get clean
+
+# Download & compile dependencies. Doing this separately allows caching this
+# layer when no dependencies are updated.
```

Comment:
nit: comment isn't quite right. It won't cache in between CI runs having equal dependencies, but will cache between different built images of the _same_ CI run.

Reply:
Yeah, I've been looking into CI layer caching too. It looks like it's possible with GitHub Actions' caching APIs. I'll be trying that out when I start working with CI.